src/XMakeBuildEngine/UnitTestsPublicOM/Definition/Project_Tests.cs

```diff
@@ -2415,6 +2416,251 @@ public void ImportPropertyEvaluatingToInvalidPath()
             }
            );
         }
+
+        [Fact]
+        public void GetItemProvenanceOnlyStringLiteral()
+        {
+            var project =
+                @"<Project ToolsVersion='msbuilddefaulttoolsversion' DefaultTargets='Build' xmlns='msbuildnamespace'>
+                  <ItemGroup>
+                    <A Include=`1;2;3`/>
+                    <B Include=`1;2;3` Exclude=`1`/>
+                    <C Include=`2;3` Exclude=`2`/>
```

Comment:
Right now it's the former, without any real cause other than it felt "good at the time". I'll run it with CPS.